objects/Player/Collision_PowerUp.gml

```diff
@@ -4,9 +4,11 @@
 //Set playerstate to corresponding state to the powerup type
 if (other.powerType == powerUps.shield) {
 	setState(playerStates.invincible);
+	audio_play_sound(sound_invinicible, 10, 0);
```

Comment:
I think we should consider a shorter sound here or we can quiet the main music for the duration. These should also be put in the set state function of their corresponding states. This is less of an issue as the powerups cannot occur back to back but its a best practice.